gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Jan 24 16:13:08 EST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
```

Comment:
I am guessing you want to rebase back to 6.7, correct?

Reply:
6.7 appears to break our gradle build scans sometimes. Only sometimes. So, I'm just going to do that in a separate PR.

Reply:
I've documented the issues in https://broadworkbench.atlassian.net/browse/DR-1438